docs/guides/docs/running-a-node/running-a-local-node.mdx

```diff
@@ -68,7 +66,7 @@ To run a local node with persistence a chain configuration file is required.
 To start the node, run the following command:
```

Comment:
The section above need to be updated since the coins array is now inside the `state_config.json`